src/Messages/OWSMessageSender.m

```diff
@@ -659,6 +631,8 @@ - (TOCFuture *)sendMessageFuture:(TSOutgoingMessage *)message
         thread:thread
         attempts:OWSMessageSenderRetryAttempts
         success:^{
+            DDLogInfo(@"Marking group message as sent to recipient: %@", recipient.uniqueId);
+            [message updateWithSentRecipient:recipient.uniqueId];
```

Comment:
We now do some bookkeeping to track which group members have received a given message.